app/controllers/general.php

```diff
@@ -984,7 +990,9 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
 
         $type = $error->getType();
 
-        $output = ((App::isDevelopment())) ? [
+        // TODO filter out secrets and server details when not in development
```

Comment:
Dev key should not affect error handler, this comments might become confusing, but even with dev key we shouldn't expose any sensitive error information.